Procfile

```diff
@@ -1 +1 @@
-worker: node src/spawn_tests.js
+worker: npm run compile && node lib/main.js test
```

Comment:
Also I guess it's a bit confusing to call it "test" right? That's what I would normally think of as testing rather than what runs in production.